docs/source/reference/yaml-spec.rst

```diff
@@ -222,6 +223,71 @@ You can also specify a specific region, zone, or Kubernetes context.
     infra: k8s/my-h100-cluster-context
 
 
+.. _yaml-spec-resources-autostop:
+
+``resources.autostop``
+~~~~~~~~~~~~~~~~~~~~~~
+
+Autostop configuration (optional).
+
+Controls whether and when to automatically stop or tear down the cluster after it becomes idle. See :ref:`auto-stop` for more details.
+
+Format:
+
+- ``true``: Use default idle minutes (5)
+- ``false``: Disable autostop
+- ``<num>``: Stop after this many idle minutes
+- ``<num><unit>``: Stop after this much time
+- Object with configuration:
+  - ``idle_minutes``: Number of idle minutes before stopping
+  - ``down``: If true, tear down the cluster instead of stopping it
+
+``<unit>`` can be one of:
+
+- ``s``, ``sec``: seconds
+
+- ``m``, ``min``: minutes
+
+- ``h``, ``hr``: hours
+
+- ``d``, ``day``: days
+
+.. maybe instead of warning, have it be a note?
+.. warning::
+
+  The time will be rounded up to the nearest minute. For example, ``10m30s`` will be rounded to ``11m``.
```

Comment:
What about `10m10s` - does it get rounded up or down? If down, we should reword this to just say `rounded to the nearest minute` (delete `up`)

Reply:
hm, its not rounded, its ceilinged so i wrote rounded up. i need to think of something more clear than rounded up then